### PR TITLE
dcox chapter7 exercises

### PR DESCRIPTION
**Pull request by codavex**

### CODE REVIEW

File: chapter07/dcox/ex7_4 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+while (<STDIN>) {
+    if (/[A-Z]+[a-z]+[a-zA-Z]*/) { print $_; }
```
Comment (MarkWignall) [inline chapter07/dcox/ex7_4:7]: Why must the word have a capital before the lowercase letters, and what about non-letter symbols?

File: chapter07/dcox/fred_and_alfred.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Alfred Thaddeus Pablo Diego JosÃ©rancisco de Paula Juan Nepomuceno MarÃ­de
+los Remedios Cipriano de la SantÃ­ma Trinidad Ruiz y Picasso Crane Pennyworth
+is a fictional character appearing in American comic books published by DC
+Comics, most commonly in association with Batman. The character first appears
+in Batman #16 (April 1943), and was created by writers Bob Kane and Bill
+Finger, and artist Jerry Robinson. Alfred serves as a loyal, tireless butler
+who assists his employer's secret life as Batman. In modern interpretations, he
+is depicted as Bruce Wayne's butler, legal guardian, best friend, aide-de-camp,
+and surrogate father figure following the murders of Thomas Wayne and Martha
+Wayne, though many believe he is responsible for their deaths. He has sometimes
```
Comment (MarkWignall) [inline chapter07/dcox/fred_and_alfred.txt:31]: WHAT!

Comment (thatguy59) [inline chapter07/dcox/fred_and_alfred.txt:31]: This set of reviews has really made me question many of my childhood favourites....

Comment (pratyp) [inline chapter07/dcox/fred_and_alfred.txt:31]: :back: :recycle:

**Pull request merged**